workloads/whereabouts/run.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash -e
+
+set -e
+
+ES_SERVER=${ES_SERVER=https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com}
+LOG_LEVEL=${LOG_LEVEL:-info}
+if [ "$KUBE_BURNER_VERSION" = "default" ]; then
+    unset KUBE_BURNER_VERSION
+fi
+KUBE_BURNER_VERSION=${KUBE_BURNER_VERSION:-1.1.0}
+CHURN=${CHURN:-false}
+WORKLOAD='node-density'
```

Comment:
This should be something more specific like `whereabouts-node-density`

Reply:
This is passed straight to kube-burner-ocp, so it would fail since there is no workload with that name. I was keeping it as is for simplicity.

Reply:
well you are modifying the job so we need some way to index the results accordingly. We should think how we can tell our index scripts this was a snowflake/multus test ;)

Reply:
I was just thinking if this can go as a separate canned workload into `kube-burner/kube-burner-ocp` repo and just have the driver script here if planning to run through e2e (i.e for CI)? That way it will be cleaner and shouldn't run into any indexing issues. Also if considering to add this into `kube-burner/kube-burner-ocp` repo, @mrbojangles3 
 can you check if we can reuse the existing `node-density` yaml resources using symbolic links or absolute paths in this workload's config? just a thought 🤔